internal/emulator/start.go

```diff
@@ -37,14 +39,34 @@ var Cmd = &cobra.Command{
 }
 
 func configuredServiceKey(
-	_ bool,
-	_ crypto.SignatureAlgorithm,
-	_ crypto.HashAlgorithm,
+	init bool,
+	sigAlgo crypto.SignatureAlgorithm,
+	hashAlgo crypto.HashAlgorithm,
 ) (
 	crypto.PrivateKey,
 	crypto.SignatureAlgorithm,
 	crypto.HashAlgorithm,
 ) {
+	if init {
+		if sigAlgo == crypto.UnknownSignatureAlgorithm {
+			sigAlgo = emulator.DefaultServiceKeySigAlgo
+		}
+
+		if hashAlgo == crypto.UnknownHashAlgorithm {
+			hashAlgo = emulator.DefaultServiceKeyHashAlgo
+		}
+
+		p, err := project.Init(sigAlgo, hashAlgo)
+		if err != nil {
+			util.Exitf(1, err.Error())
+		} else {
+			err = p.Save(project.DefaultConfigPath)
+			if err != nil {
+				util.Exitf(1, err.Error())
+			}
+		}
+	}
+
 	proj, err := project.Load(util.ConfigPath)
```

Comment:
This line uses `util.ConfigPath`, but above uses `project.DefaultConfigPath`. Are those guaranteed to be the same?

Also I think you can avoid loading this if it's been newly initiated above.